.github/workflows/k8s-ci.yml

```
@@ -23,4 +23,4 @@ jobs:
           nix-shell shell.nix --run "echo"
       - name: BootStrap k8s cluster
         run: |
-          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --label --install-zfs --install-lvm"
+          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --label-io-engine --mayastor --zfs --lvm"
```

suggestion: --mayastor could imply the labelling with a negative flag to no label, but whatever you think makes easier to use

```suggestion
          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --mayastor --zfs --lvm"
```